contracts/FIND.cdc

```diff
@@ -1741,6 +1869,28 @@ pub contract FIND {
 
 	}
 
+	access(contract) fun checkMerchantAddress(_ merchAccount: Address) {
+		// If only find can sign the trxns and call this function, then we do not have to check the address passed in.  
+		// Otherwise, would it be wiser if we hard code the address here? 
+
+		if FIND.account.address == 0x097bafa4e0b48eef {
+		// This is for mainnet
+			if merchAccount != 0x097bafa4e0b48eef {
```

Comment:
These merch addresses needs to be changed once we have them, but this is good for now